.github/actions/check_for_stc/action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Ensure STC is installed with Pyccel'
+inputs:
+  not_editable:
+    description: 'not_editable installation'
+runs:
+  using: "composite"
+  steps:
+    - name: check stc installation
+      shell: python
+      run: |
+        import pyccel, os
+        from os.path import exists, join
+        pyccel_loc = pyccel.__path__[0]
+        print(pyccel_loc)
+        print("${{ inputs.not_editable }}")
+        assert(exists(join(pyccel_loc,'extensions/STC/include/stc')))
+        if "${{ inputs.not_editable }}" != "-e":
```

Comment:
Ideally I don't think you should mention `-e` in this file as there are probably other ways of handling editable installations so it would be good to use something more general like you had before